runtime/vm/jvminit.c

```diff
@@ -1952,7 +1952,7 @@ IDATA VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved) {
 			 */
 			if (J9_ARE_ANY_BITS_SET(vm->vmRuntimeStateListener.idleTuningFlags, J9_IDLE_TUNING_GC_ON_IDLE | J9_IDLE_TUNING_COMPACT_ON_IDLE)) {
 				BOOLEAN idleGCTuningSupported = FALSE;
-#if (defined(LINUX) && (defined(J9HAMMER) || defined(J9X86))) || defined(J9ZOS39064)
+#if (defined(LINUX) && (defined(J9HAMMER) || defined(J9X86) || defined(S390) || defined(LINUXPPC64) || (defined(LINUXPPC64) && defined(J9VM_ENV_LITTLE_ENDIAN)))) || defined(J9ZOS39064)
```

Comment:
`(defined(LINUXPPC64) && defined(J9VM_ENV_LITTLE_ENDIAN))` is redundant considering `defined(LINUXPPC64)` is also added

Reply:
Yes, corrected it.